README.md

```diff
@@ -403,6 +444,231 @@ You can use `urlEndpoint` prop in an individual `IKImage` to change url for that
+
+### Basic resizing examples
+
+```js
+<IKContext urlEndpoint="https://ik.imagekit.io/your_imagekit_id">
+  // Video from related file path with no transformations - https://ik.imagekit.io/your_imagekit_id/default-video.mp4
+  <IKVideo
+    path="/default-video.mp4"
+  />
+
+  // Video resizing - https://ik.imagekit.io/your_imagekit_id/tr:w-h-300,w-400/default-video.mp4
+  <IKVideo
+    path="/default-video.mp4"
+    transformation={[{
+      height:300,
+      width:400
+    }]}
+  />
+
+  // Loading video from an absolute file path with no transformations - https://www.custom-domain.com/default-video.mp4
+  <IKVideo
+    src="https://www.custom-domain.com/default-video.mp4"
+  />
+
+  // Using a new tranformation parameter which is not there in this SDK yet - https://ik.imagekit.io/your_imagekit_id/tr:custom-value/default-video.mp4
+  <IKVideo
+    path="/default-video.mp4"
+    transformation={[{
+      custom: 'value'
+    }]}
+  />
+</IKContext>
+```
+
+The `transformation` prop is an array of objects. Each object can have the following properties.
+
+```js
+// It means first resize the video to 400x400 and then rotate 90 degree
+transformation = [
+  {
+    height: 400,
+    width: 400,
+    rt: 90
+  }
+]
+```
+
+See the complete list of transformations supported in ImageKit [here](https://docs.imagekit.io/features/video-transformation). The SDK gives a name to each transformation parameter e.g. `height` for `h` and `width` for `w` parameter. It makes your code more readable. If the property does not match any of the following supported options, it is added as it is.
+
+### List of supported transformations
+<details>
+<summary>Expand</summary>
+
+| Supported Transformation Name | Translates to parameter |
+|-------------------------------|-------------------------|
+| height | h |
+| width | w |
+| aspectRatio | ar |
+| quality | q |
+| crop | c |
+| cropMode | cm |
+| focus | fo |
+| format | f |
+| radius | r |
+| background | bg |
+| border | b |
+| rotation | rt |
+| blur | bl |
+| named | n |
+| trimmingStartOffset | so |
+| trimmingEndOffset | eo |
+| trimmingDuration | du |
+| layering | l |
+| layerInfo | i |
+| layerXPosition | lx |
+| layerYPosition | ly |
+| layerRelativePosition | lfo |
+| layerStartTime | lso |
+| layerDuration | ldu |
+| layerEndtime | leo |
+| imageOverlayWidth | w |
+| imageOverlayHeight | h |
+| imageOverlayAspectRatio | ar |
+| imageOverlayCropMethod | c |
+| imageOverlayCromMode | cm |
+| imageOverlayRelativeFocus | fo |
+| imageOverlayBorder | b |
+| imageOverlayBGColor | bg |
+| imageOverlayRadius | r |
+| imageOverlayRotationDegree | rt |
+| layerTextWidth | w |
+| layerTextFontSize | fs |
+| layerTextFontFamily | ff |
+| layerTextColor | co |
+| layerTextInnerAlignment | ia |
+| layerTextPadding | pa |
+| layerTextAlpha | al |
+| layerTextTypography | tg |
+| layerTextBGColor | bg |
+| layerTextRadius | r |
+</details>
+
+### Lazy loading videos
```

Comment:
No such thing was mentioned in the requirement.

Reply:
Fixed.